.github/scripts/check-cargo-toml-version-bumped.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+if [ -z "$(git diff HEAD origin/main -- src/)" ]; then
+  echo "No changes in the code."
+  exit 0
+fi
+
+if [ -z "$(git diff HEAD origin/main -- Cargo.toml | grep '^+version =')" ]; then
```

Comment:
You don't need the `[ -z` or making this into a string, `grep` returns an error if it didn't match, so just the two piped commands are enough as the if condition.

Reply:
Good point, I'll remove `-z`.